src/entities/Place/model.ts

```diff
@@ -548,4 +556,20 @@ export default class PlaceModel extends Model<PlaceAttributes> {
 
     return results[0].total
   }
+
+  /**
+   * For reference: https://www.evanmiller.org/how-not-to-sort-by-average-rating.html
+   * We're calculating the lower bound of a 95% confidence interval
+   */
+  static calculateLikeScoreStatement(
```

Comment:
Just to clarify: The generalization can be avoided because it uses the same params always but it's not tied to the model, it depends on other calculated fields from `UserLikesModel`. 

I'm removing parameters and hardcoding them